### PR TITLE
Added some error handling for webpack and added support for es6 webpack config

### PR DESCRIPTION
**Pull request by yachaka**
Title is self-explanatory !
Only few lines added helping for debugging ;)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -122,6 +123,12 @@ function run(input, flags, showHelp) {
 
 	runWebpack(webpackConfig).then(
 		function(webpackRes) {
+			if (webpackRes.errors.length > 0) {
+				for (var i = 0; i < webpackRes.errors.length; i++) {
+					console.error(webpackRes.errors[i]);
+				}
+				return;
```
Comment (thrandre) [inline index.js:130]: You should also terminate the process with an error code to ensure that the run returns unsucessful.

**Comment by thrandre** [comment]
Thank you! Make sure to terminate the process on webpack errors and I will merge this :)

**Comment by yachaka** [comment]
Thanks for merging and for finishing the dirty work 👍

**Comment by thrandre** [comment]
No worries! I had to fix some other issues as well. Many thanks for your contribution :+1:

**Pull request merged**